app/views/editions/secondary_nav_tabs/_related_external_links.html.erb

```diff
@@ -9,30 +9,40 @@
           empty = render(partial: "secondary_nav_tabs/related_external_links/add-another_fieldset", locals: { index: 0, id: nil })
         else
           items = @edition.artefact.external_links.each_with_index.map do | external_link, index |
+            fields = render(partial: "secondary_nav_tabs/related_external_links/add-another_fieldset", locals: { index:, id: external_link.id })
+            destroy_checkbox = if current_user.has_editor_permissions?(@resource)
```

Comment:
Small query: do we need to put a condition on `destroy_checkbox` here? May have missed something but this seems to apply the same condition twice (including the broader one below).